app/views/Main.js

```diff
@@ -1,12 +1,11 @@
-import { useNavigation } from '@react-navigation/native';
```

Comment:
It might be confusing to have Main.js at the root of the views directory while it is only used for GPS.

Reply:
agree, though im concerned with merge conflicts on the gps team.  We can move it and see what happens i suppose.